chronosphere/resource_log_allocation_config.go

```diff
@@ -104,3 +123,18 @@ func allocationFromModel(a *models.LogAllocationConfigAllocation) *intschema.Log
 		PercentOfLicense: a.PercentOfLicense,
 	}
 }
+
+func prioritiesFromModel(a *models.LogAllocationConfigHighLowPriorities) *intschema.LogPrioritiesSchema {
+	if a == nil {
+		return nil
+	}
+	return &intschema.LogPrioritiesSchema{
+		HighPriorityFilter: sliceutil.Map(a.HighPriorityFilters, searchFilterFromModel),
+		LowPriorityFilter:  sliceutil.Map(a.LowPriorityFilters, searchFilterFromModel),
+	}
+}
+
+func searchFilterFromModel(p *models.Configv1LogSearchFilter,
+) intschema.LogSearchFilterSchema {
```

Comment:
given that this is only used in a list, it's not as important, but for consistency and safety, should we have `if p == nil { return nil }` here?